### PR TITLE
Remove `uses-permission` from AndroidManifest.xml

### PR DESCRIPTION
**Pull request by briankabiro**
Finishes #34

### CODE REVIEW

**Comment by briankabiro** [comment]
@mikehardy. I guess this should be good to go as it's a duplicate of #35.

File: README.md [hunk 1]
```diff
@@ -61,57 +66,57 @@ Add permissions to your `android/app/src/main/AndroidManifest.xml` file.
 ...
 ```
 
+## Upgrading to 2.x
+
+You need to add permissions manually. `react-native-get-sms-android` does not automatically require permissions from 2.x. Refer to [this](https://github.com/briankabiro/react-native-get-sms-android/issues/34) issue.
 
+You need to require permissions based on what functions you plan to use:
```
Comment (mikehardy) [inline README.md]: Suggest you mention where they need to be added?
```suggestion
You need to require permissions in your `AndroidManifest.xml` file's `application` element based on what functions you plan to use like [the official documentation](https://developer.android.com/guide/topics/permissions/overview) describes:
```

Just being mindful of future support requests ;-)

**Comment by mikehardy** [comment]
Yeah it looked good - I added one comment that was just puffing up the documentation, not altering anything. But could go as is even. Cheers Brian

**Comment by briankabiro** [comment]
Awesome, thanks.

Committed it. Going to merge it now.

**Pull request merged**